lib/wallaby/dsl/helpers.ex

```diff
@@ -0,0 +1,43 @@
+defmodule Wallaby.DSL.Helpers do
+  alias Wallaby.Session
+
+  def take_screenshot(session) do
+    image_data =
+      Session.request(:get, "#{session.base_url}session/#{session.id}/screenshot")
```

Comment:
It doesn't need to happen in this PR but we should really abstract this away somehow.